tools/loader/Loader.cpp

```diff
@@ -123,6 +124,19 @@ llvm::cl::opt<bool> dumpGraphOpt("dumpGraph",
                                  llvm::cl::desc("Prints Graph to stdout"),
                                  llvm::cl::cat(modelExportCat));
 
+llvm::cl::list<std::string> doNotConvertNodesOpt(
```

Comment:
we already have `do-not-quantize-node` option, wondering if we could merge those to a single option.
As in principal those are the nodes we do not touch during any kind of processing.

Reply:
I was wondering about this too. The only reason I didn't do it, is because I couldn't come up with a good name :).
Any idea?

Reply:
Maybe something like `keep-original-precision`?

Reply:
Went with `keep-original-precision-for-nodes`.